save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -277,7 +277,7 @@ class VulnerabilityController(
     ): Mono<StringResponse> = vulnerabilityDateDto.toMono()
         // TODO: add permission check
```

Comment:
btw, speaking about permissions, have we limited interactions with vulnerabilities to owner and superadmins?

Reply:
It still needs to be refactored